e2e/pages/app/input-dropdown/input-dropdown.testpage.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="row">
+    <div class="col-md-5">
+        <form [formGroup]="form" novalidate>
+            <div class="form-group m-nil">
+
+        <ux-input-dropdown
+            [(selected)]="selected"
+            id="dropdown"
+            [dropdownOpen]="dropdownOpen"
+            [disabled]="disabled"
+            (dropdownOpenChange)="dropdownOpenChange($event)"
+            [(filter)]="filter"
+            [allowNull]="allowNull"
+            [maxHeight]="maxHeight"
+            [placeholder]="placeholder"
+            aria-label="Filter input"
+            formControlName="inputForm">
```

Comment:
Most of these inputs are not used for the tests. Try to keep test pages as simple as possible for the tests that are being added. This makes it easier to diagnose if a test starts to fail. In this case perhaps you only need
```html
<ux-input-dropdown
  id="dropdown"
  [placeholder]="placeholder"
  [disabled]="disabled">
```